components/MobileNavigation.js

```diff
@@ -20,23 +20,53 @@ export default class MobileNavigation extends Component {
 
   state = {
     showMobileNav: false,
-    expandMobileNav: false
+    expandMobileNav: false,
+    nowSelectMenu: null,
+    scrollPosition: null
   }
 
   componentDidMount () {
+    const { navs } = this.props
     window.addEventListener('scroll', this.handleScroll)
+    const scrollPosition = navs.reduce((obj, nav, index) => {
+      const navScroll = {
+        name: nav.label,
+        scroll: document.getElementById(nav.label.toLowerCase()).getBoundingClientRect().top + window.scrollY
+      }
+      obj[index] = navScroll
+      return obj
+    }, [])
+    this.setState({
+      scrollPosition: scrollPosition
```

Comment:
Since `scrollPosition` is not used in `render()`, there is no need to store it inside state.

Instead, you can just store it as an instance attribute: `this.navigationTargets = navigationTargets`.